doc/source/install/upgrade_guide.rst

```diff
@@ -45,18 +45,18 @@ To upgrade the cluster, follow these steps for each node:
 
      .. code-block:: bash
 
-        sudo yum remove 'Percona*'
-        sudo yum install Percona-XtraDB-Cluster-57
+        $ sudo yum remove 'Percona-XtraDB-Cluster-56*'
+        $ sudo yum install Percona-XtraDB-Cluster-57
 
    * On Debian or Ubuntu:
 
      .. code-block:: bash
 
-        sudo apt-get remove percona-*
-        sudo apt-get install percona-xtradb-cluster-57
+        $ sudo apt-get remove percona-xtradb-cluster-56*
+        $ sudo apt-get install percona-xtradb-cluster-57
 
 #. Back up :file:`grastate.dat`, so that you can restore it
-   if it is corrupted or zeroed out due to network issue, for example.
+   if it is corrupted or zeroed out due to network issue.
 
```

Comment:
For debian/ubuntu you'll need to manually stop the service after upgrade because service is automatically started during the install/upgrade process.